chaos-bugbounty-list.json

```diff
@@ -195,7 +195,11 @@
         "dingtalk.com",
         "alibaba-inc.com",
         "aliyun-inc.com",
-        "alimama.com"
+        "alimama.com",
+	"amap.com",
+	"ele.me",
+	"youku.com",
+	"koubei.com"
```

Comment:
These domains are not listed at https://hackerone.com/alibaba?type=team, can you mention the source information? @j14ncn